samconfig.toml

```diff
@@ -0,0 +1,10 @@
+version = 0.1
+
+[dev-oskwil.deploy.parameters]
+stack_name = "oskwil-lev"
```

Comment:
Hmm - should this be committed or ignored?

Reply:
committed, we want to track this